packager/react-packager/src/Server/index.js

```diff
@@ -291,7 +291,8 @@ class Server {
       // Clear cached bundles in case user reloads
       this._clearBundles();
       this._hmrFileChangeListener(absPath, this._bundler.stat(absPath));
-      return;
+      // when HMR clients connected, also allow live reloading clients
+      // return;
```

Comment:
> I think this is the single required change to make this work but I'm not sure what would be the best way to support this feature without affecting HMR performance.

The changes of other two files(RCTDevMenu.m and DevServerHelper.java) avoid the clients process both `onchange` and HMR, because clients can enable live reloading and hot reloading.

Thanks for your point, I tested competition, HMR first and Live Reload first, every are not good, just worked.

Is it possible to bundle once for Live Reload and HMR? or more common works to reduce bundle time? I can't handle this topic, I don't know details of how packager/bundler works yet.